app/src/userpages/components/StreamPage/Show/PartitionsView/index.jsx

```diff
@@ -0,0 +1,94 @@
+// @flow
+
+import React, { useCallback, useState } from 'react'
+import { connect } from 'react-redux'
+import { I18n } from 'react-redux-i18n'
+import { Link } from 'react-router-dom'
+import cx from 'classnames'
+
+import type { Stream } from '$shared/flowtype/stream-types'
+import type { StoreState } from '$shared/flowtype/store-state'
+import { updateEditStream } from '$userpages/modules/userPageStreams/actions'
+import { selectEditedStream } from '$userpages/modules/userPageStreams/selectors'
+import TextInput from '$shared/components/TextInput'
+
+import styles from './partitionsView.pcss'
+import routes from '$routes'
+
+type OwnProps = {
+    disabled: boolean,
+}
+
+type StateProps = {
+    stream: ?Stream,
+}
+
+type DispatchProps = {
+    updateEditStream: (Stream) => void,
+}
+
+type Props = OwnProps & StateProps & DispatchProps
+
+function PartitionsView(props: Props) {
+    const { stream, updateEditStream, disabled } = props
+    // $FlowFixMe
+    const { partitions } = stream
+
+    const [value, setValue] = useState(String(partitions))
+
+    const onCommit = useCallback(() => {
+        let numberValue = Number.parseInt(value, 10)
+        // if entered value is NaN use existing value
+        numberValue = Number.isNaN(numberValue) ? partitions : numberValue
+        setValue(String(numberValue))
+        updateEditStream({
+            ...stream,
+            partitions: numberValue,
+        })
+    }, [stream, updateEditStream, partitions, value])
+
+    const onChange = useCallback((event) => {
+        setValue(event.target.value)
+    }, [setValue])
+
+    return (
+        <div className={styles.root}>
+            <p className={styles.description}>
+                {I18n.t('userpages.streams.partitionsDescription')}
+                <Link to={`${routes.docsStreams()}#partitioning`}>
```

Comment:
you can rebase on `development` to get the updated Docs links, it will be something like, `routes.docsStreamsPartitioning`

Reply:
👍 thanks